pom.xml

```diff
@@ -231,6 +249,28 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>exec-maven-plugin</artifactId>
+        <version>3.0.0</version>
+        <executions>
+          <execution>
+            <id>split-oapi</id>
+            <phase>prepare-package</phase>
+            <goals>
+              <goal>java</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <mainClass>com.redhat.cloud.oapi_splitter.OApiSplitter</mainClass>
```

Comment:
The same for this, the ``oapi_splitter`` package naming style (underscore) was reserved in the Java spec for domain names and is otherwise not encouraged. Should have been ``com.redhat.cloud.oapi.splitter.`` for example.